tests/upsert_validator/conftest.py

```diff
@@ -7,45 +7,6 @@
 from bigchaindb.upsert_validator import ValidatorElection
 
 
-@pytest.fixture
-def b_mock(b, network_validators):
```

Comment:
why have these been moved?

Reply:
I needed them for some tests that were more appropriately stored in `tests/commands/` (ie testing the execution of `upsert-validator foo` on the command line), so I moved the fixtures so I'd have access to them across all the test directories.

Reply:
Could you please point out where the following fixtures are being used in your tests i.e. `b_mock`, `valid_election`, `mock_get_validators`?

Reply:
I ended up building my own elections, so I guess I can move this back